### PR TITLE
Added functions for transaction DAL

### PR DESCRIPTION
**Pull request by fichlar**

### CODE REVIEW

File: backend-app/src/data-access-layer/transaction-repository.js [hunk 1]
```diff
@@ -4,20 +4,78 @@ module.exports = function({ databaseInit }) {
...
+
+        const transaction = {
+            userID: userID,
+            chargerID: chargerID,
+            meterStop: MeterStartValue
+        }
 
         databaseInit.Transactions.create(transaction)
-            .then(a => callback([], transaction.transactionID))
+            .then(a => callback([], true))
```
Comment (HugoMartinsson) [inline backend-app/src/data-access-layer/transaction-repository.js:42]: Didn´t we decide that it should return the transactionId and not a bool?

File: backend-app/src/data-access-layer/transaction-repository.js [hunk 2]
```diff
@@ -4,20 +4,78 @@ module.exports = function({ databaseInit }) {
...
+    }
+    exports.updateTransactionPayment = function(transactionID, paymentID){
+        databaseInit.Transactions.update({
+            paymentID: paymentID
+        }, {
+            where: { transactionID: transactionID },
+            returning: true,
+            raw: true
+        })
+        .then(a => callback([], true))
```
Comment (HugoMartinsson) [inline backend-app/src/data-access-layer/transaction-repository.js:56]: Should this return the entire transaction instead?

File: backend-app/src/data-access-layer/transaction-repository.js [hunk 3]
```diff
@@ -4,20 +4,78 @@ module.exports = function({ databaseInit }) {
...
+    }
+    exports.updateTransactionMeter = function(transactionID, meterValue, callback){
+        databaseInit.Transactions.update({
+            meterValue: meterValue
+        }, {
+            where: { transactionID: transactionID },
+            returning: true,
+            raw: true
+        })
+        .then(a => callback([], true))
```
Comment (HugoMartinsson) [inline backend-app/src/data-access-layer/transaction-repository.js:72]: and this one aswell

**Review by HugoMartinsson — CHANGES_REQUESTED** [review]
Didn´t we decide to change the return values to not be bool except for delete-functions? See comments!

**Review by HugoMartinsson — APPROVED** [review]

**Review by EricSoder — APPROVED** [review]

**Pull request merged**